catalog/app/utils/AWS/S3.js

```diff
@@ -124,7 +125,15 @@ function useSmartS3() {
           params = null
         }
 
+        let forceProxy = false
+        if (params && 'forceProxy' in params) {
```

Comment:
`'forceProxy' in params` instead of `params?.forceProxy` because you need to check the prototype chain?